src/vs/workbench/contrib/chat/browser/actions/chatQuickInputActions.ts

```diff
@@ -93,8 +94,9 @@ class AskQuickQuestionAction extends Action2 {
 		});
 		disposableStore.add(openInChat);
 		disposableStore.add(openInChat.onChange(async () => {
-			await this._currentSession?.openInChat(this._input!.value);
+			await this._currentSession?.openInChat(this._lastAcceptedQuery ?? this._input!.value);
```

Comment:
You could avoid the ! if you set a `input` variable to the outer function scope

Reply:
will add in debt week